src/layout.rs

```diff
@@ -310,6 +310,20 @@ impl Layout {
         Ok(())
     }
 
+    pub(crate) fn find_editor_tagged(&self, tag_char: char) -> Option<CanonicalizedPath> {
+        self.background_suggestive_editors
+            .iter()
+            .find_map(|(_, editor)| {
+                let suggestive_editor = editor.borrow();
+                let editor = suggestive_editor.editor();
+                if editor.tag() == Some(tag_char) {
+                    editor.path().clone()
+                } else {
+                    None
+                }
```

Comment:
Can be shortened using [`then_some`](https://doc.rust-lang.org/std/primitive.bool.html#method.then_some).